webapp-django/crashstats/topcrashers/tests/test_views.py

```diff
@@ -0,0 +1,223 @@
+                    'facets': {
+                        'signature': [{
+                            'term': u'FakeSignature1 \u7684 Japanese',
+                            'count': 100,
+                            'facets': {
+                                'platform': [{
+                                    'term': 'WaterWolf',
+                                    'count': 50,
+                                }],
+                                'is_garbage_collecting': [{
+                                    'term': 't',
+                                    'count': 50,
+                                }],
+                                'hang_type': [{
+                                    'term': 1,
+                                    'count': 50,
+                                }],
+                                'process_type': [{
+                                    'term': 'plugin',
+                                    'count': 50,
+                                }],
+                            }
+                        }]
+                    },
+                    'total': 250
+                }
+            else:
+                # Return results for the current week.
+                results = {
+                    'hits': [],
+                    'facets': {
+                        'signature': [{
+                            'term': u'FakeSignature1 \u7684 Japanese',
+                            'count': 100,
+                            'facets': {
+                                'platform': [{
+                                    'term': 'WaterWolf',
+                                    'count': 50,
+                                }],
+                                'is_garbage_collecting': [{
+                                    'term': 't',
+                                    'count': 50,
+                                }],
+                                'hang_type': [{
+                                    'term': 1,
+                                    'count': 50,
+                                }],
+                                'process_type': [{
+                                    'term': 'plugin',
+                                    'count': 50,
+                                }],
+                            }
+                        }]
+                    },
+                    'total': 250
+                }
+
+            results['hits'] = self.only_certain_columns(
+                results['hits'],
+                params['_columns']
+            )
+            return results
+        supersearch().get.side_effect = mocked_supersearch_get
+
+        # first without a version
+        no_version_url = self.base_url + '?product=WaterWolf'
+        url = self.base_url + '?product=WaterWolf&version=19.0'
+        reports_count_default = url
+        reports_count_100 = (
+            self.base_url + '?product=WaterWolf&version=19.0&_facets_size=100'
+        )
+
```

Comment:
Unless there's a really obvious reason for doing it this way, it's it better to let `self.client` take care of turning these variables into query strings? E.g. `self.client.get(self.base_url, {'product': 'WaterWolf'})`